app/Entirety/smartdatamodels/forms.py

```diff
@@ -0,0 +1,49 @@
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import Submit
+from django import forms
+
+from smartdatamodels.models import SmartDataModel
+
+
+class SmartDataModelForm(forms.ModelForm):
+    schema_link = forms.URLField(required=False)
+    jsonschema = forms.JSONField(required=False)
+
+    def __init__(self, view_only: bool = True, *args, **kwargs):
+        super(SmartDataModelForm, self).__init__(*args, **kwargs)
+
+        self.helper = FormHelper(self)
+
+        if not view_only:
```

Comment:
I don't find the Save button even if the user is not "viewer". May be put default value of `view_only` to `False `here because you are only setting this value when it is True from `views.py`.

Reply:
![image](https://github.com/N5GEH/n5geh.tools.entirety/assets/79838286/c568a6c4-1dd1-43e0-8c27-8bc0c7189650)

Reply:
It should work now